administrator/components/com_content/models/article.php

```diff
@@ -730,6 +730,13 @@ public function shareTokenGenerate()
 	 */
 	public function shareToken($articleId)
 	{
+		//checking for valid articleId
+		if ($articleId == 0)
+		{
+			$this->setError(JText::_('COM_CONTENT_INVALID_ARTICLEID'));
```

Comment:
@nikitadhiman setError() is deprecated, why do you use it? You need to throw an exception here.